tests/bud.bats

```diff
@@ -1662,12 +1772,14 @@ load helpers
 
 @test "bud using gitrepo and branch" {
   target=gittarget
+  # FIXME: this test takes a really long time. Is it necessary to do twice?
```

Comment:
almost 4 minutes, because it uses fedora as base image and runs a dnf install. Is there some other simpler test we can do instead? Can we create a new module under `github.com/containers` just for testing this? Or even just a side branch of `containers/BuildSourceImage` that just does `FROM scratch` and `RUN echo got here`?

Reply:
How about switching it to:
run_buildah bud --format docker --layers -f tests/bud/shell/Dockerfile -t test git://github.com/containers/buildah#master

Reply:
Runs in 3.5 seconds.